docs/update-cli.md

```diff
@@ -0,0 +1,26 @@
+# Update OpenAPI CLI
+
+If you've already installed OpenAPI CLI, you'll want to keep it as up to date as possible. Staying on top of versions ensures that you get the latest functionality and bug fixes. Depending on how you originally installed OpenAPI CLI, there are two ways to update it.
+
+:::success Tip
+Before you begin, ensure that you have the latest version of npm installed.
+:::
+
+## How to update your openapi-starter installation
```

Comment:
I don't think this heading is correct. The openapi-starter is a template. Any repository can have a package installation.

> ## How to update a project-level installation